packages/sdks-tests/src/e2e-tests/editing.spec.ts

```diff
@@ -332,13 +343,19 @@ test.describe('Visual Editing', () => {
         model: 'page',
       });
 
-      await item1.click();
+      await expect(
+        page.frameLocator('iframe').getByText('Item 1')
+      ).toBeVisible({ timeout: 5000 });
+      
+      // Re-query the item1 element as it might have been recreated
+      const updatedItem1 = page.frameLocator('iframe').getByText('Item 1');
+      await updatedItem1.click();
```

Comment:
we want to avoid hardcoded timeouts unless there's a special reason for them. Playwright has automated waiting and retrying so that we don't need to provide hardcoded timeout values all the time. Also, in this case, the default is already 5000 so this does nothing: https://github.com/BuilderIO/builder/blob/b351bd82895c40394e9c022af2c9762cbb1a8add/packages/sdks-tests/playwright.config.ts#L69

you also don't need to redefine the locator twice:
```suggestion
      // Re-query the item1 element as it might have been recreated
      const updatedItem1 = page.frameLocator('iframe').getByText('Item 1');
      await expect(updatedItem1).toBeVisible();      
      await updatedItem1.click();
```